test/goss.yaml

```diff
@@ -9,6 +9,17 @@ command:
     timeout: 1000
 {{end}}
 
+command:
+  ps -ef|grep -v grep|grep java:
+    exit-status: 0
+    stdout:
+{{if getEnv "BROKER_CONFIG_GLOBAL_MAX_SIZE" }}
+    - "-Dbrokerconfig.globalMaxSize={{ .Env.BROKER_CONFIG_GLOBAL_MAX_SIZE }}"
```

Comment:
Oh! now I see why the echo before. 

I'd rather check the file content themselves. Dgoss might be tedious, if you don't want to check the file, check for the stdout and I'll change it to file later.

Reply:
this is not related to echo above.  This is needed to confirm that the arguments are being properly translated and passed to the command line.  Checking the file only is not sufficient as arguments are translated at that time.  This test works fine and I don't feel it should be changed.